pkg/build/api/v1/types.go

```diff
@@ -349,11 +361,8 @@ type GitBuildSource struct {
 	// ref is the branch/tag/ref to build.
 	Ref string `json:"ref,omitempty" protobuf:"bytes,2,opt,name=ref"`
 
-	// httpProxy is a proxy used to reach the git repository over http
-	HTTPProxy *string `json:"httpProxy,omitempty" protobuf:"bytes,3,opt,name=httpProxy"`
-
-	// httpsProxy is a proxy used to reach the git repository over https
-	HTTPSProxy *string `json:"httpsProxy,omitempty" protobuf:"bytes,4,opt,name=httpsProxy"`
+	// proxyConfig defines the proxies to use for the git clone operation
+	ProxyConfig `json:",inline" protobuf:"bytes,3,opt,name=proxyConfig"`
```

Comment:
Did we ever ship HTTPProxy and HTTPSProxy in the previous release?  I.e. did we break the api between 1.3 and 1.4 here?